.github/workflows/label.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler
+
+name: Labeler
+on: [pull_request]
```

Comment:
The sample uses `[pull_request_target]`, you may need to use that for permission reasons?

Reply:
That's weird. I think I created this file through the github wizard. From the docs, it look like [`pull_request_target`](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request_target) is the better choice.

Reply:
fixed